scripts/build-glean-metadata.py

```diff
@@ -18,6 +18,21 @@ def _serialize_sets(obj):
     return obj
 
 
+def _normalize_metrics(name):
+    # replace . with _ so sirv doesn't think that
+    # a metric is a file
+    metric_name = name.replace(".", "_")
+
+    # if a metric name starts with "metrics", uBlock Origin
+    # will block the network call to get the JSON resource
+    # See: https://github.com/mozilla/glean-dictionary/issues/550
+    # To get around this, we add "data" to said metric names
+    if metric_name.split("_")[0] == "metrics":
+        return f"data_{metric_name}"
```

Comment:
It's probably easier to just prefix everything with `data_`. That way we don't need conditional logic here.